adam-core/src/main/java/org/bdgenomics/adam/io/FastqRecordReader.java

```diff
@@ -206,20 +217,19 @@ protected final int positionAtFirstRecord(final FSDataInputStream stream,
         Text buffer = new Text();
         long originalStart = start;
         
-        // Advance to the start of the first record that ends with /1
-        // We use a temporary LineReader to read lines until we find the
-        // position of the right one.  We then seek the file to that position.
-        stream.seek(start);
         LineReader reader;
         if (codec == null) {
+	    // Advance to the start of the first record that ends with /1
+	    // We use a temporary LineReader to read lines until we find the
+	    // position of the right one.  We then seek the file to that position.
+	    stream.seek(start);
             reader = new LineReader(stream);
         } else {
             // see above note about 
             // SplittableCompressionCodec.createInputStream needing the stream
             // to be at offset 0
-            stream.seek(0);
```

Comment:
I think this is still useful info to keep around, but I'll update the comment to better reflect the changed code.